src/marqo/core/utils/special_characters_encoding.py

```diff
@@ -0,0 +1,7 @@
+import re
+
+def custom_encode(s: str) -> str:
```

Comment:
Can you provide some doc strings for these functions?

Reply:
Added!